config_defaults_inc.php

```diff
@@ -2085,6 +2085,28 @@
 $g_bug_closed_status_threshold = CLOSED;
 
 /**
+ * Bug becomes closed for submission of automated bug reports (attaches)
+ * if its status is >= $g_bug_collect_reports_threshold. The bug starts 
+ * accept more bug reports again if its status becomes less than this 
+ * threshold. The default is ASSIGNED. 
+ * Read-only, closed and fixed bugs are closed for bug report submission 
+ * regardless of this option.
+ * See also $g_bug_max_auto_reports option.
+ * See also SOAP.mc_issue_report function.
+ * @global integer $g_bug_collect_reports_status_threshold
+ */
+$g_bug_collect_reports_status_threshold = ACKNOWLEDGED;
```

Comment:
Your P.S. is not clear and should be clarified in the documentation of the config option.

You opposite of your argument is also true.  You can imagine assigning the issue for investigation, and then you no longer receiving reports, though receiving reports may be good because you gather more information about who is hitting the issue.

In addition to the max number of reports per issue.  I wonder if we should have a max # of reports by a crash reporting user per hour.  That is to protect the system against getting a ton of reports to quickly, specially if there is a bug with alias calculation (e.g. exception message is included in hash and it has a non-unique value in it for each user).